debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/transforms/type/deserializer/TypeDeserializer.java

```diff
@@ -16,5 +17,9 @@ public Object deserialize(AbstractType<?> abstractType, ByteBuffer bb) {
         return abstractType.getSerializer().deserialize(bb);
     }
 
+    public Object deserialize(AbstractType<?> abstractType, ComplexColumnData ccd) {
+        return null;
```

Comment:
How comes null is returned here? I think it warrants a comment if not even a warning in the log if it'd e.g. imply a specific column type wouldn't be captured (I'm not quite clear about the implications).

Reply:
Added a comment, this method has been overwritten in `MapTypeDeserialize`, `SetTypeDeserializer` and `ListTypeDeserializer`, and won't be called for the deserialization for any non-complex abstract types.

Reply:
> How comes null is returned here? I think it warrants a comment if not even a warning in the log if it'd e.g. imply a specific column type wouldn't be captured (I'm not quite clear about the implications).

Added a comment, this method has been overwritten in `MapTypeDeserialize`, `SetTypeDeserializer` and `ListTypeDeserializer`, and won't be called for the deserialization for any non-complex abstract types.

Reply:
Ok, thx. Perhaps throw an exception even then? Or perhaps push this method to a subclass/interface where it always does make sense?

Reply:
Thanks, much better now!